src/puppetlabs/puppetdb/utils.clj

```diff
@@ -237,6 +237,14 @@
    :prefix "/pdb/cmd"
    :version (or version :v1)})
 
+(defn pdb-meta-base-url
+  [host port & [version]]
+  {:protocol "http"
+   :host host
+   :port port
+   :prefix "/pdb/meta"
+   :version (or version :v1)})
```

Comment:
I was just following the 4 or 5 other functions in this file that do the same thing with a different :prefix. Are you thinking we shouldn't have that on any of them?